rootfs/etc/templates/haproxy/haproxy.tmpl

```diff
@@ -512,6 +512,9 @@ backend {{ $backend.ID }}
 {{- range $e1 := short 10 $allow.Exception }}
     acl allow_exception_src{{ $i }} src{{ range $e := $e1 }} {{ $e }}{{ end }}
 {{- end }}
+{{- with $allow.SourceHeader }}
```

Comment:
Change to `if` instead just to follow the convention.

Reply:
done :+1: